PharmacyApi/Utilities/ServiceResult.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace PharmacyApi.Utilities
+{
+    public class ServiceResult<T>
+    {
+        public T? Result { get; set; }
+        public bool Success { get; set; }
```

Comment:
Boolean - Specifying Boolean field names as questions is best practice: "IsSuccess"
Some old timers like to call it "SuccessFlag" which is fine, if you still use a washboard and clothesline in your backyard...